File: Client/Client/Assets/Scripts/JsonScripts/ReconnectJson.cs
using System;

/*Nicht Benutzbar wegen JsonContainer*/

[Serializable]
public class ReconnectJson
{
    public ContainerJson matchStart;
    public ContainerJson snapshot;
    public ContainerJson next;

    //public ReconnectJson(ContainerJson matchStart, ContainerJson snapshot, ContainerJson next)
    //{
    //    this.matchStart = matchStart;
    //    this.snapshot = snapshot;
    //    this.next = next;
    //}


    //public override string ToString()
    //{
    //    return "ReconnectJson{" +
    //            "matchStart=" + matchStart +
    //            ", snapshot=" + snapshot +
    //            ", next=" + next +
    //            '}';
    //}
}

File: Server/team05_server_source/Server/src/main/java/de/team05/server/json/ReconnectJson.java
package de.team05.server.json;

public class ReconnectJson {
    ContainerJson matchStart;
    ContainerJson snapshot;
    ContainerJson next;

    public ReconnectJson(ContainerJson matchStart, ContainerJson snapshot, ContainerJson next) {
        this.matchStart = matchStart;
        this.snapshot = snapshot;
        this.next = next;
    }

    @Override
    public String toString() {
        return "ReconnectJson{" +
                "matchStart=" + matchStart +
                ", snapshot=" + snapshot +
                ", next=" + next +
                '}';
    }
}
